### PR TITLE
Add deprecation warning for alpha.service-controller.kubernetes.io/exclude-balancer in v1.16 CHANGELOG

### PR DESCRIPTION
**Pull request by andrewsykim**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
In https://github.com/kubernetes/kubernetes/pull/80238, we deprecated the node label `alpha.service-controller.kubernetes.io/exclude-balancer` in favor of `node.kubernetes.io/exclude-balancer`. The CHANGELOG should reflect this change under "Deprecations and Removals". 

Also removes extra white spaces in the CHANGELOG. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: CHANGELOG-1.16.md [hunk 1]
```diff
@@ -236,6 +236,7 @@ The main themes of this release are:
   - Aggregated discovery requests can now timeout. Aggregated API servers must complete discovery calls within 5 seconds (other requests can take longer). Use the feature gate `EnableAggregatedDiscoveryTimeout=false` to temporarily revert behavior to the previous 30 second timeout if required (the temporary `EnableAggregatedDiscoveryTimeout` feature gate will be removed in v1.17). ([#82146](https://github.com/kubernetes/kubernetes/pull/82146), [@deads2k](https://github.com/deads2k))
   - the `scheduler.alpha.kubernetes.io/critical-pod` annotation is removed. Pod priority (`spec.priorityClassName`) should be used instead to mark pods as critical. ([#80342](https://github.com/kubernetes/kubernetes/pull/80342), [@draveness](https://github.com/draveness))
   - the NormalizeScore plugin set is removed from scheduler framework config API. Use ScorePlugin only. ([#80930](https://github.com/kubernetes/kubernetes/pull/80930), [@liu-cong](https://github.com/liu-cong))
+  - the node label `alpha.service-controller.kubernetes.io/exclude-balancer` which excludes a node from cloud LoadBalancers (using Service Type=LoadBalancer) is deprecated in favor of `node.kubernetes.io/exclude-balancer`
```
Comment (yastij) [inline CHANGELOG-1.16.md]: @andrewsykim - can we add the previsional removal release ?

File: CHANGELOG-1.16.md [hunk 2]
```diff
@@ -381,6 +381,7 @@ The main themes of this release are:
...
   - the `scheduler.alpha.kubernetes.io/critical-pod` annotation is removed. Pod priority (`spec.priorityClassName`) should be used instead to mark pods as critical. ([#80342](https://github.com/kubernetes/kubernetes/pull/80342), [@draveness](https://github.com/draveness))
   - the NormalizeScore plugin set is removed from scheduler framework config API. Use ScorePlugin only. ([#80930](https://github.com/kubernetes/kubernetes/pull/80930), [@liu-cong](https://github.com/liu-cong))
+  - the node label `alpha.service-controller.kubernetes.io/exclude-balancer` which excludes a node from cloud load balancers (using Service Type=LoadBalancer) is deprecated in favor of `node.kubernetes.io/exclude-balancer`. Support for `alpha.service-controller.kubernetes.io/exclude-balancer` will be removed in v1.18.
```
Comment (liggitt) [inline CHANGELOG-1.16.md]: hmm... we don't usually count retconned release notes as effective deprecation notices

Comment (andrewsykim) [reply CHANGELOG-1.16.md]: Not sure I follow, we already deprecated this in https://github.com/kubernetes/kubernetes/pull/80238, just updating the release notes to reflect that change. The original PR mentions v1.17 for removal, but in https://github.com/kubernetes/kubernetes/pull/82855 we updated that to v1.18.

Comment (andrewsykim) [reply CHANGELOG-1.16.md]: I agree it's not really effective, but the deprecation already happened in https://github.com/kubernetes/kubernetes/pull/80238  and the release notes don't reflect that change, shouldn't we update it anyways?

Comment (andrewsykim) [reply CHANGELOG-1.16.md]: We can log a warning in service controller for this but it will be very verbose

**Pull request merged**